### ISSUE TITLE
Test set queries' ground truth

### ISSUE THREAD

**Issue by mint44** [scenenn/shrec17#2]
Can you upload the ground truth of snn.* queries that come from test set. The current "queries.csv" file in evaluation code just contains samples from train set and validation set. Thank you so much.

**Comment by pqhieu** [comment]
Hi @ngocminhbui,

I have uploaded the ground truth for test set. You can find it in at `evaluation/test.csv`
